docker-database/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+  db:
+    container_name: database
```

Comment:
```suggestion
```

better to let docker-compose to name the container. It is a deterministic name

Reply:
removed